ironfish/src/defaultNetworkDefinitions.ts

```diff
@@ -54,120 +54,4 @@ export const TESTING = `{
   }
 }`
 
-export const MAINNET = `
```

Comment:
Can we bring main net back so we can use it as a staging ground for the main net parameters?

Reply:
Ahh, ok yea I wasn't about a case where we'd use the consensus params but not the genesis block.